packages/epetra/test/VbrMatrix/cxx_main.cpp

```diff
@@ -1781,7 +1781,7 @@ int checkMergeRedundantEntries(Epetra_Comm& comm, bool verbose)
   //FillComplete, which internally calls MergeRedundantEntries, the
   //matrix should contain 1.0 in each entry.
 
-  for(int i=myFirstRow; i<=myLastRow; ++i) {
+  for(unsigned int i=myFirstRow; i<=myLastRow; ++i) {
```

Comment:
`unsigned` shouldn't be the right type for a global index in Epetra.

Reply:
The `unsigned int i` felt rather wrong, but seemed to do the trick. I suppose I was _hoping_ large scale edge cases were tested for.
I went back and found if I changed `int myLastRow` and `int col` from the `unsigned int` I set to `long long`. Then I could return to `int i`. That seems better.
I'm still looking for an alternative to the `if ((usedLength+numPositions) < allocatedLength)` `strict-overflow` one though.

Reply:
> I suppose I was _hoping_ large scale edge cases were tested for.

That's ... optimistic.

Reply:
> I went back and found if I changed `int myLastRow` and `int col` from the `unsigned int` I set to `long long`. Then I could return to `int i`. That seems better.

Local row indices have type `int` in Epetra.

Reply:
> I'm still looking for an alternative to the `if ((usedLength+numPositions) < allocatedLength)` strict-overflow one though.

That's a tricky one.  I wonder if the compiler has figured out that two of those variables are actually the same thing.